src/helper/containsSqlFunction.ts

```diff
@@ -27,7 +27,7 @@ export function containsSqlFunction(
   return (
     expression.parsedSql.type === 'function' &&
     lowerCaseFunctionNames.includes(
-      (expression.parsedSql as SqlFunction).functionName.toLowerCase(),
+      (expression.parsedSql as SqlFunction).functionName.toString().toLowerCase(),
```

Comment:
this should be `(expression.parsedSql as SqlFunction).getEffectiveFunctionNam().toLowerCase()` more future proof